### DISCUSSION TITLE
The maintainer required!!

### DISCUSSION THREAD

**Discussion by HenilMistry** [NetworkingDevs/NS3-GUI-HELPER#7] [General]
# Maintainer Requires!!
As you might know that we have hosted the site of this project on [Github](https://networkingdevs.github.io/NS3-GUI-HELPER/) and maintaining that site with latest updates of this project is very essential because for any project, there must be common platform where you get all the information about how to access and use that application! So, keeping that thing in the mind and the importance of maintenance of site we need a maintainer person who might help to maintain the site branch!

## Role and Responsibilities
- Website management
- Content writing
- Branch Management (Only site branch, where web-site is hosted)

## Prerequisite
- HTML, CSS
- MARKDOWN File Management
- GitHub Website Templates
- GitHub Basics
- Good Communication

If you think you can help this project grow and have all what is takes to be a maintainer then contact us on "**_henilmistry74496@gmail.com_**"

**Comment by 20CE034** [comment]
@HenilMistry Please don't reject me! I am a good developer!! perhaps check the pull request regarding "Latest release update in hosted GitHub site!" also all the best for project!

**Reply by HenilMistry** [reply]
# Got it
- Hey @20CE034 Thanks a lot for your contribution to maintain the site!
- Totally appreciate your efforts and hoping for more of it soon!
